tensorflow/core/kernels/tensor_array.h

```diff
@@ -35,7 +35,14 @@ limitations under the License.
 namespace tensorflow {
 
 typedef Eigen::ThreadPoolDevice CPUDevice;
+
+#if GOOGLE_CUDA
```

Comment:
That's weird because I have seen a lot of typedef for GPUDevice using #if GOOGLE_CUDA so I was wondering if it was a missing one